app/assets/stylesheets/page.css.scss

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
Looks good. Maybe better to call it "admin.css.scss", or probably even better, make an admin folder in stylesheets and put this in there.